faq/java.inc

```diff
@@ -128,3 +128,22 @@ C/Fortran code).
 We suggest users experiencing performance problems to contact the
 <a href=\"https://www.open-mpi.org/community/lists/ompi.php\">
 OMPI users mailing list</a>.";
+
+/////////////////////////////////////////////////////////////////////////
+
+$q[] = "What are the known limitations of the Java Bindings?";
+$anchor[] = "java_limitations";
+
+$a[] = "There are currently issues with Intel Truescale (PSM) and Omnipath
```

Comment:
@jsquyres Do you have a suggestion for a better place to put the known issues section?  I can reword it to not use "currently", but this is intended to be a temporary warning for any users seeing this issue.